### ISSUE TITLE
Add parameter to specify timezone

### ISSUE THREAD

**Issue by dabukalam** [nuxeh/url-bot-rs#14]
Ability to force a specific timezone for db logging... Or could also just add/remove hours when printing previous posts

**Comment by nuxeh** [comment]
Indeed. The `time` crate is used to get the time/date, which should be able to handle timezones, in fact it uses `to_local` at the moment.

https://github.com/nuxeh/url-bot-rs/blob/master/src/sqlite.rs#L27

It seems that it has a `tzset` function, which sadly isn't documented right now... https://docs.rs/time/0.1.40/time/fn.tzset.html

**Comment by dabukalam** [comment]
Source for that doesn't look very implemented... https://docs.rs/time/0.1.40/src/time/lib.rs.html#284-287

Maybe this instead... https://docs.rs/chrono/0.4.6/chrono/offset/struct.Utc.html

**Comment by nuxeh** [comment]
https://github.com/rust-lang-deprecated/time/blob/master/README.md says "In
case you're looking for something a little fresher and more actively
maintained have a look at the `chrono` crate."

So you might be right...
